image/templates/common/ca-setup.sh

```diff
@@ -57,14 +57,14 @@ function create_directory {
 	local dir="$1"
 	echo "The following certificates will be used as additional CAs:"
 	from_file_args=()
-	for f in $dir/*.crt; do
+	for f in $dir/*.crt $dir/*.pem; do
```

Comment:
Since we're touching this line...
```suggestion
	for f in "$dir"/*.crt "$dir"/*.pem; do
```